tests/test_adjoint_jacobian.py

```diff
@@ -816,9 +821,12 @@ def circuit(params):
     params = np.random.rand(n_params)
 
     qnode_kokkos = qml.QNode(circuit, dev_kokkos, diff_method="adjoint")
+    qnode_cpu = qml.QNode(circuit, dev_default, diff_method="parameter-shift")
```

Comment:
Could you please update all `_cpu` variables to `_default`? If sensible, of course.

Reply:
sure!